warehouse/templates/accounts/invite-confirmation.html

```diff
@@ -0,0 +1,37 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "confirm-action.html" %}
+
+{% block title %}
+    {% trans %}Confirm Invite{% endtrans %}
+{% endblock %}
+
+{% block main %}
+
+<form method="POST">
+  <div class="form-group">
+    <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+    <span class="form-group__text">
+      {% trans project_name=project_name, role_name=desired_role, project_link=request.route_url('packaging.project', name=project_name) %}
+      Would you like to accept this invitation to join '<a href="{{ project_link }}">{{ project_name }}</a>' as a project {{ role_name }}?
+      {% endtrans %}
+    </span>
+  </div>
+  <div>
+    <input name="accept" type="submit" value="{% trans %}Accept{% endtrans %}" class="button button--primary">
+    <input name="decline" type="submit" value="{% trans %}Decline{% endtrans %}" class="button button--danger">
```

Comment:
```suggestion
    <input name="decline" type="submit" value="{% trans %}Decline{% endtrans %}" class="button">
```
Very small nit here but this button is not actually "dangerous". We want to be careful how much we use the "danger" color.

Reply:
I was attracted to the red 😜